build.gradle

```diff
@@ -1,7 +1,32 @@
+buildscript {
```

Comment:
I don't like the reordering - I preferred to start the file with the definition of the project's group and version.

Reply:
I _believe_ that `buildscript` must come before `plugins {}` block because of how the script gets processed, but I can take another crack at it

Reply:
I don't know much about Gradle, but I'm under the impression it worked well before. :wink:

Reply:
I just tested it with

```
group 'org.junit-pioneer'
version '0.1-SNAPSHOT'

buildscript {
    repositories {
        mavenLocal()
        maven {
            name "GradlePlugins"
            url 'https://plugins.gradle.org/m2/'
        }
        maven {
            name "MavenSnapshot"
            url 'https://oss.sonatype.org/content/repositories/snapshots/'
        }
    }

    dependencies {
        classpath group: 'com.diffplug.gradle.spotless', name: 'spotless', version: '2.4.1'
        classpath group: 'org.junit.platform', name: 'junit-platform-gradle-plugin', version: '1.0.0'
        classpath group: 'com.squareup.okhttp3', name: 'okhttp', version: '3.9.0'
        classpath group: 'org.ajoberstar', name: 'grgit', version: '2.0.1'
    }
}

plugins {
    id 'com.github.ben-manes.versions' version '0.15.0'
}
```

and it fails with 

```

* Where:                                             
Build file '/home/mkobit/Workspace/personal/junit-pioneer/junit-pioneer/build.gradle' line: 25            

* What went wrong:                                   
Could not compile build file '/home/mkobit/Workspace/personal/junit-pioneer/junit-pioneer/build.gradle'.  
> startup failed:                                    
  build file '/home/mkobit/Workspace/personal/junit-pioneer/junit-pioneer/build.gradle': 25: only buildscript {} and other plugins {} script blocks are allowed before plugins {} blocks, no other statements are allowed                                                
                                                     
  See https://docs.gradle.org/4.2/userguide/plugins.html#sec:plugins_block for information on the plugins {} block                                                                                                  
                                                     
   @ line 25, column 1.                              
     plugins {                                       
     ^                                               
                                                     
  1 error                                            


* Try:                                               
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.                                                                                                      

* Get more help at https://help.gradle.org   
```